notifier/registrator.go

```diff
@@ -40,52 +40,60 @@ const (
 	mattermostSender  = "mattermost"
 )
 
-// RegisterSenders watch on senders config and register all configured senders
+// RegisterSenders creates all senders and registers them.
 func (notifier *StandardNotifier) RegisterSenders(connector moira.Database) error { //nolint
```

Comment:
and can return error...

Reply:
I think testing this func is overengineering. If we test this func, then we increment some coverage results, but what is the profit? Do we increase reliablitiy? I think no, because every senders consctructor is tested in place. To test `RegisterSenders` func, we have to cover N cases (for each Sender), but it is repeating of constructors tests, so it is redundant